xpra/codecs/nvenc/encoder.pyx

```diff
@@ -1668,7 +1813,7 @@ cdef class Encoder:
         profile = os.environ.get("XPRA_NVENC_PROFILE", "")
         profile = os.environ.get("XPRA_NVENC_%s_PROFILE" % csc_mode, profile)
         #now see if the client has requested a different value:
-        profile = options.strget("h264.%s.profile" % csc_mode, profile)
+        profile = options.get("h264.%s.profile" % csc_mode, profile)
```

Comment:
`strget` is needed because we get `options` from the client, through a packet encoder which may mangle bytes and strings. (#3229)

Reply:
I'm not sure exactly what is going on here but months ago using `strget` caused the nvenc encoder to exception out on initialization where I was running it, so I had to add a manual hack changing it to `get` just to get it to work in the first place without crashing on me and disabling nvenc entirely. Currently I'm using the `nvidia/cudagl:11.0-base-ubuntu20.04` docker container, however I believe this was the case with just a vanilla ubuntu 20.04